python/tests/test_minecraft.py

```diff
@@ -30,6 +30,7 @@ def newMineOp(inventory):
         lambda obj, tool: mine_op(inventory, obj, tool),
         unwrap=False)
 
+@unittest.skip("no reason")
```

Comment:
Is this test ignored by accident? I tried locally and it passes, but `test_minecraft_planning_with_abstractions` has no asserts inside.
```suggestion
```

Reply:
Oops, sorry. I turned this test off while playing with Minimal MeTTa (because it was taking too much time), and occasionally added this change to this PR.